jena-core/src/main/java/org/apache/jena/irix/IRIProviderAny.java

```diff
@@ -21,7 +21,7 @@
 import java.util.Objects;
 
 /**
- * Non-resolving provider that accepts anything string for <...>
+ * Non-resolving provider that accepts anything string for &lt;...&gt;.
```

Comment:
Unrelated to jena-jdbc - another javadoc issue that checking the build showed up.